instaloader/instaloadercontext.py

```diff
@@ -210,8 +210,8 @@ def login(self, user, passwd):
         # Override default timeout behavior.
         # Need to silence mypy bug for this. See: https://github.com/python/mypy/issues/2427
         session.request = partial(session.request, timeout=self.request_timeout) # type: ignore
-        session.get('https://www.instagram.com/web/__mid/')
-        csrf_token = session.cookies.get_dict()['csrftoken']
+        res = session.get('https://www.instagram.com/accounts/login/')
+        csrf_token = re.findall(r"csrf_token\":\"(.*?)\"", res.text)[0]
```

Comment:
```suggestion
        csrf_json = self.get_json('accounts/login/', {}, session=session)
        csrf_token = csrf_json['config']['csrf_token']
```

Reply:
just tried out this suggestion, works fine for me as well!

this is my first pull request, so no clue on what to do now (gitlab user normally)? Feel free to recommend the next steps :-D